version.txt

```diff
@@ -1 +1 @@
-84.0.0-beta.1
+84.0.0-beta.2
```

Comment:
I added this change since treescript didn't manage to handle it, yesterday in https://firefox-ci-tc.services.mozilla.com/tasks/YruwlHPBTeCX3ebtrQS1Vw/

Althought the error message looks unrelated. It didn't manage to push because it was trying to reach the master branch:

```
2020-11-18 18:37:06,070 - treescript.git - WARNING - No branch provided in the task payload. Staying on the default one
[...]
2020-11-18 18:40:19,010 - git.cmd - DEBUG - Popen(['git', 'rev-list', 'master@{u}..master', '--'], cwd=/app/workdir/src, universal_newlines=False, shell=None, istream=None)
```
https://firefox-ci-tc.services.mozilla.com/tasks/YruwlHPBTeCX3ebtrQS1Vw/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FYruwlHPBTeCX3ebtrQS1Vw%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive_backing.log#L54